docs/current/cli/guides/632021-proxy-requests-api.md

```diff
@@ -0,0 +1,6 @@
+---
+slug: /cli/632021/proxy-requests-api
+displayed_sidebar: 'current'
+---
+
+# Proxy Requests to the Dagger API
```

Comment:
We may cut `dagger listen`. This guide is still useful, but will rely on other commands.

Reply:
OK